docker-compose-local-genesis.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.2'
+services:
+  create-beacon-chain-genesis:
+    image: "gcr.io/prysmaticlabs/prysm/cmd/prysmctl:latest"
```

Comment:
It would be better to use some version instead of `latest`.

Reply:
yep but current stable segfaults because there are no builds for M1 and it segfaults in rosetta mode for people on apple laptops.